.travis.yml

```diff
@@ -4,10 +4,13 @@ android:
   components:
     - tools
     - platform-tools
-    - build-tools-25.0.1
-    - android-25
+    - build-tools-27.0.2
+    - android-27
     - extra
 
+before_install:
+  - yes | sdkmanager "platforms;android-27"
```

Comment:
This is to work around an issue with Travis CI: https://github.com/travis-ci/travis-ci/issues/8874

Reply:
maybe other workaround...